client_test.go

```diff
@@ -1,11 +1,10 @@
 package cohere
 
 import (
-	"os"
 	"testing"
 )
 
-var apiKey = os.Getenv("API_KEY")
+var apiKey = "qjGVNgxWsj6pHdXsprfa55Au7msQxS7Igl0Wey2n"
```

Comment:
😅

Reply:
Classic. I'm going to force push and also invalidate the API key.